apps/client/src/app/services/import-activities.service.ts

```diff
@@ -210,7 +230,7 @@ export class ImportActivitiesService {
   }
 
   private parseDate({
-    content,
+  content,
```

Comment:
❓

Reply:
this was accidental 😶, will cross check twice next time, for now I reverted it